cdxev/__main__.py

```diff
@@ -294,12 +294,14 @@ def create_validation_parser(
     parser.add_argument(
         "--filename-pattern",
         help=(
-            "Regex for validation of file name. "
-            "If no Regex is given the default 'name_version_hash_timestamp.cdx.json' "
-            "or 'bom.json' is used, "
-            "where name, version and timestamp are mandatory and taken from metadata. "
-            "Hash is optional as this is not a required information"
+            "Regex for validation of file name. If not specified, a default regex depending on "
```

Comment:
file name or filename? Both is correct, just would like some consistency, you are mostly using "filename", so I would stick to it

Reply:
Very good catch, will fix.

Thanks for generally paying close attention to documentation :pray: It's important and many reviewers skip over it.